ckanext/xloader/jobs.py

```diff
@@ -222,6 +222,11 @@ def tabulator_load():
                               resource_id=resource['id'],
                               mimetype=resource.get('format'),
                               logger=logger)
+        except rq_timeouts.JobTimeoutException as e:
```

Comment:
Wouldn't it be better to put this further out, eg line 256?

Reply:
@ThrawnCA moved out to wrap the `tabulator_load` method call with the try/catch. Lemme know if that is actually the right place. Thanks!

Reply:
It's a start, but I think line 267 would be a better place.